src/commands/Misc/Birthday/view-birthday.ts

```diff
@@ -0,0 +1,27 @@
+import { SkyraCommand } from '#lib/structures';
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import { ApplyOptions } from '@sapphire/decorators';
+import type { Message } from 'discord.js';
+
+@ApplyOptions<SkyraCommand.Options>({
+	cooldown: 5,
+	description: LanguageKeys.Commands.Misc.ViewBirthdayDescription,
+	extendedHelp: LanguageKeys.Commands.Misc.ViewBirthdayExtended,
+	runIn: ['text']
+})
+export default class extends SkyraCommand {
+	public async run(message: Message, args: SkyraCommand.Args) {
+		const user = args.finished ? message.author! : await args.pick('userName');
+		const birthDate = this.context.client.schedules.queue.find(
+			(schedule) => schedule.taskID === 'birthday' && schedule.data.guildID === message.guild!.id && schedule.data.userID === user.id
+		)?.time;
+
+		return message.send(
+			// @ts-ignore i have no idea what causes this
```

Comment:
This happens because the options in both overloads are different, you need to do

```typescript
birthDate
	? args.t(LanguageKeys.Commands.Misc.ViewBirthdaySet, { birthDate: birthDate.getTime(), user: user.tag })
	: args.t(LanguageKeys.Commands.Misc.ViewBirthdayNotSet, { user: user.tag });
```